src/test/groovy/graphql/schema/idl/SchemaParserTest.groovy

```diff
@@ -361,4 +364,120 @@ class SchemaParserTest extends Specification {
         e.errors[0].message.contains("parsing has been cancelled")
 
     }
+
+    def "correctly parses schema keyword block, include Query, does not include Mutation type"() {
+        // From RFC to clarify spec https://github.com/graphql/graphql-spec/pull/987
+        when:
+        def schema = """schema {
```

Comment:
Funky indentation is required for schema printing comparison

Reply:
comparing via strings will often result in funky indents - its the nature of the game